src/llmcompressor/transformers/finetune/text_generation.py

```diff
@@ -258,21 +255,27 @@ def main(
     for training and eval
     :param training_args: Arguments pertaining to training loop configuration
     """
+    configure_logger()
```

Comment:
why do we need to configure here when we're running on import of the logger config module?

Reply:
This is left over from attempting to get it to play correctly as a library. It's no longer needed